File: python/queries_gen/date.py
class Date: 
  
  def __init__(self, string, week): 
    # string = 2020-12-30
    # week = Wednesday
    self.year = string[0:4] # 2020
    self.month = string[5:7] # 12
    self.day = string[8:10] # 31
    self.weekDay = week[0:3] # Wed

  def toString(self):
    print("Year: ", self.year)
    print("Month: ", self.month)
    print("Day: ", self.day)
    print("Day of the week: ", self.weekDay)

    

File: c++/inheritance/date.cpp
#include "date.hh"

Date::Date(string d, string m, string y, string dow){
	this->day = d;
	this->month = m;
	this->year = y;
	this->dayOfWeek = dow;
}

void Date::toString(){
	cout << "Data: " << day <<" "<< month <<" "<< year << " - " << dayOfWeek << endl;
}


int main()
{
	Date d = Date("12", "Jen", "2021", "Wednesday");
	d.toString();
}
